dependencies/newts/pom.xml

```diff
@@ -31,11 +39,29 @@
       <artifactId>newts-cassandra-search</artifactId>
       <version>${newtsVersion}</version>
       <exclusions>
+        <exclusion>
+          <groupId>com.boundary</groupId>
+          <artifactId>high-scale-lib</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.datastax.cassandra</groupId>
+          <artifactId>cassandra-driver-core</artifactId>
+        </exclusion>
         <exclusion>
           <groupId>com.google.guava</groupId>
           <artifactId>guava</artifactId>
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.datastax.cassandra</groupId>
+      <artifactId>cassandra-driver-core</artifactId>
+      <version>${cassandraVersion}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.raccoonfink</groupId>
+      <artifactId>high-scale-lib</artifactId>
```

Comment:
how can I find what repository this is retrieved from? Something stood out about the group ID so I was curious 😉 I can't find it though

Reply:
:D

It's in the (currently flagged as phising) opennms repo.opennms.org site, under https://repo.opennms.org/3rdparty/com/raccoonfink/high-scale-lib/

The upstream `high-scale-lib` includes overridden versions of some JDK built-in collections, which conflicts with modern JDKs. I just forked it and re-published with a new version, without the JDK bits, since cassandra doesn't rely on them.

If I have some time, I'll probably change this to a shaded lib that only pulls in the classes it needs, but for now this gets it working.